docs/conda.md

```diff
@@ -191,6 +195,49 @@ process hello {
 
 It is also possible to use [mamba](https://github.com/mamba-org/mamba) to speed up the creation of conda environments. For more information on how to enable this feature please refer to {ref}`Conda <config-conda>`.
 
+## Migration to Unified Package Management
+
+The unified {ref}`package-page` system provides a modern alternative to the conda directive. When the `preview.package` feature is enabled, you can use the new syntax:
+
+### Before (conda directive):
+```nextflow
+process example {
+    conda 'samtools=1.15 bcftools=1.15'
+    
+    script:
+    """
+    samtools --version
+    bcftools --version
+    """
+}
+```
+
+### After (package directive):
+```nextflow
+process example {
+    package 'samtools=1.15 bcftools=1.15', provider: 'conda'
+    
+    script:
+    """
+    samtools --version
+    bcftools --version
+    """
+}
+```
+
+To enable the unified package system:
+
+```groovy
+// nextflow.config
+nextflow.preview.package = true
+```
+
+The unified system provides:
+- Consistent interface across different package managers
+- Plugin-based architecture for extensibility
+- Better integration with containerization platforms
+- Support for multiple package managers (conda, pixi, etc.)
```

Comment:
Think we should be explicit about which package managers are supported. ideally also pak or install2.r for R

Reply:
Agreed! Do you just want to support the package managers that wave supports?

I also thought this might be a nice abstraction for the core container directives, but I wanted to keep the scope tight.